### ISSUE TITLE
[3.1] Downloading by taxonomic name does not produce expected file

### ISSUE THREAD

**Issue by tito-abbo** [Symbiota/Symbiota#1470]
### Contact Details

tabbo002@ucr.edu

### Symbiota portal name

CCH2

### What happened?

I searched the portal for all _Arctostaphylos _rudis collected from 2018 to 2024. This returned only 36 records. When I tried to download them I got a file of some 25 thousand+ random records rather than what I searched for. I tried as compressed and non compressed, Drawincore, symbiota native, the two available encodings, etc; in all cases, I got the same 25k+ lines long random specimens data file. 
![image](https://github.com/BioKIC/symbiota-docs/assets/72320523/d6e31f13-2328-40c7-b284-d2fb42439f2a)
![image](https://github.com/BioKIC/symbiota-docs/assets/72320523/a80e1aba-9281-481f-86fa-4afb3159222f)
![image](https://github.com/BioKIC/symbiota-docs/assets/72320523/6f87b643-db3a-41a9-9247-d5d97a856a6d)
![image](https://github.com/BioKIC/symbiota-docs/assets/72320523/0ff946be-abc9-4175-aaf6-46ef8cea056a)

### What operational systems are you seeing the problem on?

Linux

### What browsers are you seeing the problem on?

Firefox, Brave

**Comment by themerekat** [comment]
Mitch Cruzan reports that "I'm trying to download all records for Achillea millefolium" and the download is not working.

**Comment by themerekat** [comment]
I did some testing.
The correct file is returned when you do a locality or spatial search, just not when you do a taxon-based search.

**Comment by tito-abbo** [comment]
> I did some testing. The correct file is returned when you do a locality or spatial search, just not when you do a taxon-based search.

Can confirm. The taxonomic fields seem to be ignored. I tried adding SLO and SB counties to my search as a work around but I just go a bunch of data for those counties i.e. not just the taxa I indicated.  

I also tried the search from the classic search interface and that time the download took over 10 minutes and yielded an empty file that said, "the query failed to resolve".
![image](https://github.com/BioKIC/Symbiota/assets/72320523/65d2b370-1b24-4f27-9769-7fd19e4e28ab)
